docker/docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: '3'
 services:
   graph-node:
     image: graphprotocol/graph-node
+    user: "1000:1000"
```

Comment:
also not sure this is going to work until the upstream build was produced by the file above